models/completion/fact_completions.sql

```diff
@@ -48,8 +48,14 @@ select
         when scaled_progress >= 0.1 and scaled_progress < 0.2
         then '10-19%'
         else '0-9%'
-    end as completion_bucket
+    end as completion_bucket,
+    username,
```

Comment:
Can you follow the pattern of the other columns and do

`users.username as username,`

for these? It makes it easier later on to be explicit in case an upstream table adds a column with one of these names.